src/Parameter/AbstractParameter.php

```diff
@@ -17,7 +17,7 @@ abstract class AbstractParameter implements ParameterInterface
 
     /**
      * The value to use.
-     * @var string
+     * @var string|array|integer|float|bool
```

Comment:
That's what we call `mixed` 😂

Reply:
Doesn't `mixed` also include objects?

Reply:
> mixed is equivalent to a Union Type of:
> string|int|float|bool|null|array|object|callable|resource

https://php.watch/versions/8.0/mixed-type

While I get your point, the API "only" accepts values of the scalar types, or arrays of scalar types: https://www.php.net/manual/en/language.types.intro.php

Would `@var scalar|scalar[]` be clearer?

I know it's not great, but short of refactoring this to different parameter interfaces like `StringParameter`, `IntegerParameter` etc I don't see a better solution. And even then there will probably be some issues with the wildcard `*` which is a string.

Reply:
It's all good, wouldn't put in more effort. It was just a joking remark :)